jobs-builder/jobs/include/os2node.yaml.inc

```diff
@@ -0,0 +1,21 @@
+{#
+# Copyright (c) 2020 Red Hat, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Convert OS name to node label string.
+#}
+{% if os == "centos8" -%}
+centos8_azure
+{% elif os == "debian10" -%}
+debian10_azure
```

Comment:
This should be gone by the end of the day, see: http://lists.katacontainers.io/pipermail/kata-dev/2021-November/002268.html

Reply:
removed